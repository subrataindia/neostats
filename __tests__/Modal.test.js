import 'react-native';
import React from 'react';
import renderer from 'react-test-renderer';
import MyModal from '../src/components/Modal';

test('Empty Modal renders correctly', () => {
  const tree = renderer
    .create(
      <MyModal
        visible={false}
        type={''}
        title={''}
        body={''}
        closable={false}
        hideCallBack={() => {}}
      />,
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
