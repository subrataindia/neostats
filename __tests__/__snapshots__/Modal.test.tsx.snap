// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Empty Modal renders correctly 1`] = `
<Modal
  animationType="slide"
  hardwareAccelerated={false}
  visible={false}
/>
`;

exports[`Error Modal renders correctly 1`] = `
<Modal
  animationType="slide"
  hardwareAccelerated={false}
  visible={false}
/>
`;

exports[`renders correctly 1`] = `
<Modal
  animationType="slide"
  hardwareAccelerated={false}
  visible={false}
/>
`;
